rust/openvasctl/src/pref_handler.rs

```diff
@@ -0,0 +1,624 @@
+// SPDX-FileCopyrightText: 2024 Greenbone AG
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+use std::collections::HashMap;
+
+use models::{ports_to_openvas_port_list, AliveTestMethods, CredentialType, Scan, Service, VT};
+use redis_storage::dberror::RedisStorageResult;
+
+use super::cmd;
+use super::openvas_redis::{KbAccess, VtHelper};
+
+const OID_SSH_AUTH: &str = "1.3.6.1.4.1.25623.1.0.103591";
+const OID_SMB_AUTH: &str = "1.3.6.1.4.1.25623.1.0.90023";
+const OID_ESXI_AUTH: &str = "1.3.6.1.4.1.25623.1.0.105058";
+const OID_SNMP_AUTH: &str = "1.3.6.1.4.1.25623.1.0.105076";
+const OID_PING_HOST: &str = "1.3.6.1.4.1.25623.1.0.100315";
+
+const BOREAS_ALIVE_TEST: &str = "ALIVE_TEST";
+const BOREAS_ALIVE_TEST_PORTS: &str = "ALIVE_TEST_PORTS";
+const ALIVE_TEST_SCAN_CONFIG_DEFAULT: u8 = 0x00;
+
+fn int_to_bool(value: u8) -> bool {
+    if value == 0 {
```

Comment:
this can be simplified to 
```
value == 0
```
instead of that if.

Reply:
I think the whole method is not necessary, as `value == 0` can be used directly where it is needed